modules/algebra/include/Gaussian3D.h

```diff
@@ -57,6 +57,12 @@ IMPALGEBRAEXPORT DenseGrid3D<float> get_rasterized(const Gaussian3Ds &gmm,
                                                    double cell_width,
                                                    const BoundingBox3D &bb);
 
+IMPALGEBRAEXPORT DenseGrid3D<float> get_rasterized_fast(const Gaussian3Ds &gmm,
```

Comment:
Are you sure you mean `float` here? IMP convention is to use `double` for floating point values (on 64-bit there's very little difference in performance). If you have a pressing need to use `float` instead here, you should add a comment explaining why.

Also, this function needs a doxygen-formatted comment explaining what it does.